srv/components/motr/init.sls

```diff
@@ -19,7 +19,7 @@
 include:
   - components.motr.prepare
   - components.motr.install
-  - components.motr.config
+  - components.motr.config.init_mod
```

Comment:
Should be components.motr.config, no?
Also, where is srv/components/motr/config/init.sls that includes the 3 config sub-modules?
We need it until we make the switch to CES workflow

Reply:
similar to above implementation